torch/_inductor/codegen/wrapper.py

```diff
@@ -1939,13 +2065,16 @@ def use_thread_local_cached_output_tensor(idx, output):
             self.wrapper_call.writeline(f"return {{{', '.join(output_refs)}}};\n}}")
 
     def generate_before_suffix(self, result):
-        if V.graph.aot_mode:
+        if V.graph.aot_mode and not V.graph.is_const_graph:
             result.writeline("} // AOTInductorModel::run_impl")
 
     def generate_end(self, result):
         if V.graph.aot_mode:
-            result.writeline("} // namespace aot_inductor")
-            result.writeline("} // namespace torch")
+            if V.graph.is_const_graph:
+                result.writeline("} // AOTInductorModel::_const_run_impl")
+            else:
+                result.writeline("} // namespace aot_inductor")
+                result.writeline("} // namespace torch")
```

Comment:
Why do we not place AOTInductorModel::_const_run_impl in the ```torch``` namespace?

Reply:
It is in the `torch` namespace. The whole logic is, if `is_const_graph`, we would generate a constant folding code. The `generate_end` for the constant folding code is the closing of that part.
If it's the main codegen, constant folding code is part of the "main", the `generate_end` for `else` is closing the `main`.